tests/tools/onert_train/src/onert_train.cc

```diff
@@ -255,14 +255,6 @@ int main(const int argc, char **argv)
       exit(-1);
     }
 
-    if (vdata_length < tri.batch_size)
```

Comment:
Q. Is the flow working as you think? 

I mean, I read below code in this file

```cpp
          const int num_valid_step = vdata_length / tri.batch_size;
          for (uint32_t n = 0; n < num_valid_step; ++n)
```

if `(vdata_len < tri.batch_size)` is true, the above code(running validation) doesn't enter the loop. I'm curious whether it's okay or not.

(IHMO, warning log is necessary)

Reply:
I updated this PR to add vdata_length value check logic to the if condition.